packages/react-ui/src/field/Select.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { animations, box, fonts, spacings } from "../theme";
+
+export const Select = ({ children, ...props }) => (
+  <StyledSelect {...props}>{children}</StyledSelect>
+);
+
+Select.propTypes = {
+  children: PropTypes.node.isRequired
+};
+
+const INPUT_HEIGHT = "5.4rem";
+
+const iconArrowDownSvg = `<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><g clip-path="url(#clip0)"><path d="M8.425 11.573l7.11-6.072a.709.709 0 00.26-.537.709.709 0 00-.26-.537.973.973 0 00-.629-.223.973.973 0 00-.63.223l-6.48 5.536-6.481-5.536a.973.973 0 00-.63-.223.973.973 0 00-.628.223.708.708 0 00-.261.537c0 .202.094.395.26.537l7.11 6.072a.91.91 0 00.29.165 1.021 1.021 0 00.68 0 .91.91 0 00.29-.165z" fill="rgb(255, 112, 103)"/></g><defs><clipPath id="clip0"><path fill="currentColor" d="M0 0h16v16H0z"/></clipPath></defs></svg>`;
```

Comment:
you should move icon into a react component and use it inside the select component this would be simpler to style

Reply:
Same as input date, i used appearance and background css rules to change arrow style

Reply:
yes I've see ;) my point is that using it css give us less control than a regular node. it was initially done like that because all the styling were done in css  files. now we have a dedicated component we could add addintional markup.